crypto/cryptlib.c

```diff
@@ -167,22 +167,6 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_list ap;
     TCHAR buf[256];
     const TCHAR *fmt;
-# ifdef STD_ERROR_HANDLE        /* what a dirty trick! */
```

Comment:
Ah, this is some Win thing... can you suggest a comment that explains this ifdef?